install/Knative-custom-install.md

```diff
@@ -145,197 +143,190 @@ service mesh. If you install any of the following options, you must install the
-| **knative/build**                                    |                   |                                                            |                 |                    |                                                                                     |
-| [`release.yaml`][3.1]                                | -                 | -                                                          | Included        | -                  | -                                                                                   |
-| **knative/eventing**                                 |                   |                                                            |                 |                    |                                                                                     |
-| [`release.yaml`][4.1]                                | _Required_        | -                                                          | -               | Included           | Includes the in-memory channel provisioner.                                         |
-| [`eventing.yaml`][4.2]                               | _Required_        | -                                                          | -               | Included           | No channel provisioner.                                                             |
-| [`in-memory-channel.yaml`][4.3]                      | _Required_        | -                                                          | -               | _Required_         | Only the in-memory channel provisioner.                                             |
-| [`kafka.yaml`][4.4]                                  | _Required_        | -                                                          | -               | _Required_         | Only the Kafka channel provisioner.                                                 |
-| **knative/eventing-sources**                         |                   |                                                            |                 |                    |                                                                                     |
-| [`release.yaml`][5.1]                                | _Required_        | -                                                          | -               | _Required_         | Source types: [Kubernetes][6], [GitHub][6.1], [Container image][6.3]                |
-| [`release-with-gcppubsub.yaml`][5.2]                 | _Required_        | -                                                          | -               | _Required_         | Source types: [Kubernetes][6], [GitHub][6.1], [Container image][6.3], [PubSub][6.4] |
-| [`message-dumper.yaml`][5.3]                         | _Required_        | -                                                          | -               | _Required_         | Event logging service for debugging.                                                |
+**Install files in bold are the recommended standard install suitable for most use cases.**
```

Comment:
Same comment as above. Also the additional bold in the table dilutes the visibility of the component names (its already busy enough).

If there was a "Serving" version of the `release.yaml` a simple statement like "Tip: Install the `release.yaml` to ensure your installation is compatible with most use cases."

If we need to address new users in the adv. install guide, those statements should be made inline, maybe as newline in the Notes column for each file? ->

\<br\>Tip: Compatible with most use cases.